content/php/concepts/math-functions/terms/bindec/bindec.md

```diff
@@ -0,0 +1,78 @@
+---
+Title: 'bindec()'
+Description: 'Returns an integer representing the decimal value of the binary string passed as input.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `bindec()` function in PHP is used to convert a binary string representation to its decimal equivalent. It takes a binary string as input and returns the corresponding decimal value.
+
+## Syntax
+
+Here's an explanation of the `bindec()` function along with syntax:
+
+```pseudo
+bindec(string $binary_string): int
+```
+
+- **Parameters**:
```

Comment:
Let's remove the `**Parameters**` and `**Return Value**` and just have the two bullet points.